src/styles/blocks/_clockwise.scss

```
@@ -0,0 +1,36 @@
+.clockwise {
+  display: flex;
+  list-style: none;
+  background-color: $color-clockwise;
+
+  @include smallCircle($size-clockwise);
+
+  &__sec {
+    position: absolute;
+    height: 50%;
+    left: 49.6%;
```

why so strange float number?
just use 
```suggestion
    left: 50%;
    transform: translateX(-50%);
```